lib/RESTClient.js

```diff
@@ -393,6 +395,53 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   send a request to get all raft sessions
+    *   get server's response and return it
+    *
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getAllRafts(reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = '/_/raft_sessions/';
+        log.debug('getting all raftSessions');
+        this.request('GET', path, log, null, null, callback);
+    }
+
+    /**
+    *   Get raft logs from bucketd
+    *   get server's response and return it
+    *
+    *   @param {string} raftId - raft session id
+    *   @param {number} [start=1] - starting sequence number
+    *   @param {number} [end=10000] - ending sequence number
+    *   @param {boolean} [targetLeader=false] - true: from leader instead of
+    *       follower
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getRaftLog(raftId, start, end, targetLeader, reqUids, callback, reqLogger) {
```

Comment:
I think it's useful to provide the `limit` parameter, because that is the preferred way for backbeat to ask that many records at most, since it does not necessarily know the start sequence , it cannot know what will be the end sequence to reach the limit.